.conform.yaml

```
@@ -0,0 +1,14 @@
+policies:
+  - type: commit
+    spec:
+      conventional:
+        types:
+          - chore
+          - ci
+          - docs
+          - refactor
+          - perf
+          - test
+          - build
```

```suggestion
          - ci
```
Hva skal `build` bety? Er det snakk om `ci/cd`, for da liker jeg å bruke `ci` i stedet. Kortere og i mine øyne mer presist.

---

Min forståelse:

- `build` er ting som omhandler bygging av appen, `mise`, `make`, `go`, etc.
- `ci` er i praksis GitHub Actions